src/state/codeSlice.ts

```diff
@@ -0,0 +1,23 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
```

Comment:
We may be better off doing one of 2 things:
1. including the code in the PlayerSlice
2. Making a new gameSlice, which holds not just the code, but also other game-relevant state

Reply:
I'd perfer #2 tbh since the code and play information could be considered as two different data classes. I can rename the codeSlice as gameSlice so that we can put in other game info in there